src/enums/nameRequestTypes.ts

```diff
@@ -0,0 +1,6 @@
+export enum NameRequestTypes {
+  BENEFIT_COMPANY = 'BC',
+  BC_COMPANY = 'CR',
+  BC_ULC_COMPANY = 'UL',
+  BC_CCC = 'CC',
+}
```

Comment:
Can we sync this list with the corp type codes (from shared corp-type-module)?

For example,
```
   BENEFIT_COMPANY = CorpTypeCd.BC_COMPANY,
   BC_COMPANY = CorpTypeCd.BC_CORPORATION,
   etc
```

Ref: https://github.com/bcgov/bcrs-shared-components/blob/972765945d09d93368753987ea697f62cfef31ff/src/modules/corp-type-module/corp-type-module.ts#L9

Reply:
I'm not sure we can, for all of them at least. Some of the codes don't exist in our `corp-type-module`, `UL` for example.

If you prefer, we can for the ones tho DO exist in both enums?

Reply:
I think (but not 100%) that, in the future, these will either all match, or should all have an equivalent.

Can you check with Thor about how we'll handle NR/corp type differences?

I'm actually trying to reduce confusion 😛 I think we should only have 1 type list, but we're not there yet and this change would be a step in that direction.

Reply:
For sure, will inquire 100%. 
That being said, if the types were in Sync we would have no need for this enum but until then, we cannot do this here.

Reply:
I believe that the CorpTypeCd enum contains all entity types as well as all NR types.

What do you mean, "cannot do this here"?

I'm hoping you can keep NameRequestTypes here but show (as per my example above) how they sync with the CorpTypeCd.